xml/yast2_sw.xml

```diff
@@ -275,6 +275,17 @@
       </para>
      </listitem>
     </varlistentry>
+    <varlistentry>
+     <term>Services</term>
+     <listitem>
+      <para>
+       Shows which packages belong to a certain service or module. Select the 
+       module or extension (for example, <literal>Basesystem</literal> or 
```

Comment:
If you apply the previous change, "module or extension" would be repeated. :-(
Change to "Select one to display"?

Reply:
I decided for 'Select one entry to display...'